### PR TITLE
one call should be better than two

### PR DESCRIPTION
**Pull request by Galithil**
this should avoid double calling the project list.

### CODE REVIEW

File: run_dir/static/js/projects.js [hunk 1]
```diff
@@ -129,13 +140,10 @@ function load_table_head(columns){
...
-function load_undefined_columns() {
-  return $.getJSON("/api/v1/projects_fields?undefined=true&project_list=" + projects_page_type, function(data) {
+// Undefined columns handled here now
+function load_undefined_columns(cols) {
     var columns_html = "";
-    $.each(data, function(column_no, column) {
+    $.each(cols, function(col_id, column) {
       $("#undefined_columns").append('<div class="checkbox">'+
           '<label>'+
             '<input type="checkbox" class="filterCheckbox" data-columngroup="undefined-columns" data-displayname="'+column+'" name="'+column+'" id="undefined-columns-'+column+'">'+
```
Comment (guillermo-carrasco) [inline run_dir/static/js/projects.js]: not using, the `col_id`, right? Just checking that you didn't typo it with `column`

Comment (Galithil) [inline run_dir/static/js/projects.js]: not using it, no, but $.each on an array gives out two members, if I only
have one, it's the index of the array, which is pretty useless

Comment (ewels) [inline run_dir/static/js/projects.js]:
```javascript
$.each(cols, function(whydoineedthisstupidindexcountihatejavascriptsomuch, column) {
  ...
});
```

Comment (Galithil) [inline run_dir/static/js/projects.js]: the other way around is a foreach in javascript, that amazingly does the
SAME BLOODY STUPID THING

**Pull request merged**